packages/api/src/webhooks/cannon.test.ts

```diff
@@ -163,6 +163,7 @@ describe("webhook cannon", () => {
       webhookServer.app.use(bodyParser.json());
       webhookServer.app.post("/webhook", (req, res) => {
         console.log("WEBHOOK WORKS , body", req.body);
+        expect(req.headers['livepeer-signature']).not.toBe(undefined);
```

Comment:
Would be nice to check if the secret actually matches what we would expect. WDYT of?
```suggestion
        const signature = req.headers['livepeer-signature']
        expect(signature).toEqual(sign(JSON.stringify(req.body), mockWebhook.sharedSecret));
```
Or maybe we should use `bodyParser.text()` instead of `json` above and avoid potential errors from mismatching `JSON.stringify` ordering here.
```suggestion
        const signature = req.headers['livepeer-signature']
        expect(signature).toEqual(sign(req.body, mockWebhook.sharedSecret));
```
(it parses the raw body bytes into a `string`)